rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc

```diff
@@ -1,27 +0,0 @@
-<?php
-
-namespace Rector\PHPUnit\Tests\PHPUnit60\Rector\ClassMethod\AddDoesNotPerformAssertionToNonAssertingTestRector\Fixture;
-
-class SkipDoTestFileInfoTest extends \Rector\Testing\PHPUnit\AbstractRectorTestCase
```

Comment:
why it removed? it is a fixture file that test skipping `AddDoesNotPerformAssertionToNonAssertingTestRector`